src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubRepositoryInfo.java

```diff
@@ -25,19 +25,18 @@
 package org.jenkinsci.plugins.github_branch_source;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
-import org.apache.commons.lang.StringUtils;
-
 import java.net.MalformedURLException;
 import java.net.URL;
+import org.apache.commons.lang.StringUtils;
 
 import static org.apache.commons.lang.StringUtils.removeEnd;
 import static org.jenkinsci.plugins.github_branch_source.GitHubSCMSource.GITHUB_COM;
 
 /**
  * Used to compute values for GitHubSCMSource from a user-specified repository URL.
- * 
- * TODO: Is it possible to compute the API URI from just a repository URL, or not because of the possibility of proxies, etc.?
- * Is it worth making a guess based on the specified host?
+ *
+ * TODO: Is it possible to compute the API URI from just a repository URL, or not because of the possibility of proxies,
+ * etc.? Is it worth making a guess based on the specified host?
```

Comment:
Breaks at an illogical point.

Reply:
:shrug: 
It considers this text, it wraps the text.